MotMystere.cpp

```diff
@@ -2,19 +2,41 @@
 //
 
 #include <iostream>
+#include <string>
+#include <ctime>
+#include <cstdlib>
+using namespace std;
+
+string getUserInput() {
+    string motMystere;
+    cout << "Entrer un mot : ";
+    cin >> motMystere;
+    cout << "Le mot est :"  << motMystere << "\n";
+    return motMystere;
+}
 
 int main()
 {
-    std::cout << "Hello World!\n";
+    string motMystere = getUserInput();
+    
+    //return 0;
+   
+    do
+    {
+        cout << "Quel est ce mot?" << endl;
+        cin >> motMystere;
+        cout << "Ce n'est pas le mot?" << endl;
+        cout << "Quel est ce mot?" << endl;
+        cin >> motMystere;
+
+    } while (motMystere !="Procra");
```

Comment:
Are you sure your algorithm is working? What happens if I give the right response on my first try, does it still tell me that I'm wrong?

Plus, isn't the user supposed to find the word supplied at the beggining of the program? If yes, I think it doesn't work here, because you always compare the user input to "Procra", no matter what the user entered at the beginning